src/phoenix/server/api/types/Project.py

```diff
@@ -33,6 +33,10 @@ def end_time(self) -> Optional[datetime]:
     def record_count(self) -> int:
         return self.project.span_count
 
+    @strawberry.field
+    def trace_count(self) -> int:
```

Comment:
thanks for this! Mind updating this to pull this value in the UI? https://github.com/Arize-ai/phoenix/blob/af90a942e6890fc1a26a88ffd71387a0215e8823/app/src/pages/projects/ProjectsPage.tsx#L129